File: day_two/addThree.js
let numOne = Number(prompt("Enter a number: "));
let numTwo = Number(prompt("Enter a second number: "));
let numThree = Number(prompt("Enter a third number: "));

let result = numOne + numTwo + numThree;

let printed = document.querySelector("p");

printed.innerHTML = result;

File: day_two/addThree.cpp
#include <iostream>

int main() {

    int num1, num2, num3, result;

    std::cout << "Enter the first number: ";
    std::cin >> num1;
    std::cout << "Enter the second number: ";
    std::cin >> num2;
    std::cout << "Enter the third number: ";
    std::cin >> num3;

    result = num1 + num2 + num3;
    std::cout << num1 << " + " << num2 << " + " << num3 << " = " << result << std::endl; 


    return 0;
}